frontend/ui/elements/font_settings.lua

```diff
@@ -6,7 +6,10 @@ local _ = require("gettext")
 --[[ Font settings for desktop linux and mac ]]--
 
 local function getUserDir()
-    return os.getenv("HOME").."/.local/share/fonts"
+    local home = os.getenv("HOME")
```

Comment:
@pazos The unit test doesn't behave exactly right for Kobo because `HOME` is actually set to `/`, but regardless it doesn't seem to make much sense to do all this unless Device:canOpenLink() is true? I adjusted `openFontDir()` accordingly.